admin/tabs/general/roles/RoleModel.ts

```diff
@@ -222,25 +262,29 @@ export class RoleModel extends HoistModel {
 
     private createGridModel(): GridModel {
         return new GridModel({
+            treeMode: true,
+            treeStyle: TreeStyle.HIGHLIGHTS_AND_BORDERS,
             autosizeOptions: {mode: 'managed'},
             emptyText: 'No roles found.',
             colChooserModel: true,
-            sortBy: 'name|asc',
+            sortBy: ['isGroupRow', 'name'],
```

Comment:
Is it necessary to sort on `isGroupRow`? That seems confusing to me

Reply:
We want the role to show on top of groups if they are on the same "level."